lib/virtfs/block_io.rb

```diff
@@ -11,6 +11,8 @@ module VirtFS
   class BlockIO
     MIN_BLOCKS_TO_CACHE = 64
 
+    attr_reader :seek_pos
```

Comment:
@roliveri the #seek method provides a mechanism to set the absolute seek position. To backup / restore the seek pointer during operations (eg push / popping the pointer), we could use relative positions, but it's often more convenient to store / use the absolute address, hence the reader.

virtfs-refs does this in a few places, when reading file / dir attributes from the fs.